src/components/Button/index.jsx

```diff
@@ -1,13 +1,26 @@
+import GoogleButtonIcon from "../GoogleButtonIcon/GoogleButtonIcon";
 import "./styles.scss";
 import PropTypes from "prop-types";
 
 const Button = (props) => {
-  return <button type={props.type} className='button'>{props.text}</button>;
+  return (
+    <button type={props.type} className={`button ${props.styleGoogle}`}>
```

Comment:
Please use [clsx](https://www.npmjs.com/package/clsx) package to handle conditional class names more elegantly